shared/js/l10n.js

```diff
@@ -1492,6 +1511,7 @@
         return getDirection(navigator.mozL10n.ctx.supportedLocales[0]);
       }
     },
+    gaiaVersion: '2.2',
```

Comment:
one other option suggested by @fabricedesre is to use navigator.userAgent and map Gecko37 to 2.2.